app/Http/Controllers/TrainingController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\MailPostTrainingRequest;
+use App\Models\Mail\Mailer;
+use App\Models\Mail\MailFactory;
+use Illuminate\Http\Request;
+
+class TrainingController extends Controller
+{
+    public function signout()
+    {
+        return view('training.signout');
+    }
+
+    public function sendsignoutmail(MailPostTrainingRequest $request) 
+    {
+        //1. validatie hier
+        $request->validate([
+            'name' => ['required', 'max:255'],
+            'date' => ['required', 'after:today']
```

Comment:
moeten we niet ook valideren of ze zich aanmelden voor een datum waarop er niks is?

Reply:
is nog niet mogelijk omdat oefensessies nog niet bestaan in de database